packages/components/src/InsightsLabel/InsightsLabel.tsx

```diff
@@ -34,7 +33,14 @@ export interface InsightsLabelProps extends LabelProps {
 
 const InsightsLabel: React.FunctionComponent<InsightsLabelProps> = ({ value = 1, text, hideIcon, className, rest, ...props }) => {
   return (
-    <Label {...rest} {...props} className={classNames(className, `ins-c-label-${value}`)} icon={!hideIcon && VALUE_TO_STATE[value].icon}>
+    <Label
+      {...rest}
+      {...props}
+      className={
+        value === 4 ? 'pf-c-label pf-m-red' : value === 3 ? 'pf-c-label pf-m-orange' : value === 1 ? 'pf-c-label pf-m-blue' : 'ins-c-label-2'
```

Comment:
Could we add that to `VALUE_TO_STATE` to remove these ternary operators?

Reply:
@rvsia Good suggestion. I will do that